web/src/lib/modals/DuplicatesSettingsModal.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+  import { Modal, ModalBody, Button, Text } from '@immich/ui';
+  import { t } from 'svelte-i18n';
+  import {
+    duplicateTiePreference,
+    findDuplicateTiePreference,
+    type SourcePreference,
+    setDuplicateTiePreference,
+  } from '$lib/stores/duplicate-tie-preferences.svelte';
+  import { mdiCogOutline } from '@mdi/js';
+
+  let { onClose }: Props = $props();
+
+  let tiePreferenceLocal = $state(duplicateTiePreference.value);
+
+  interface Props {
+    onClose: () => void;
+  }
```

Comment:
```suggestion
  type Props = {
    onClose: () => void;
  }
```
Nit: we're moving to `type`s